### DISCUSSION TITLE
Add option for bookmark to show count of documents

### DISCUSSION THREAD

**Discussion by botzill** [directus/directus#6594] [Feature Requests]
Currently when we see the bookmarks like:

<img width="212" alt="Screen Shot 2021-07-01 at 08 14 47" src="https://user-images.githubusercontent.com/4686040/124069274-93f6fa00-da44-11eb-9544-423d969a19ef.png">

You have to click on it to know the count of records it has. 

I was wondering if it's useful to have a possibility to show the count of documents on this listing, like:

```
Booked(0)
Already Booked(1)
```

This will require making a count query for all the bookmarks every time we access the page, will this impact the speed of loading the page? Maybe using cache for this?

**Comment by rijkvanzanten** [comment]
> This will require making a count query for all the bookmarks every time we access the page, will this impact the speed of loading the page? Maybe using cache for this?

Just getting the count might not be _that_ slow in general, so if we load it async and cache the result across navigations, it should be fine 🤔

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
